server/data/licenceTypes.ts

```diff
@@ -32,7 +32,7 @@ export interface BassReferral {
     addressTown?: string
     postCode?: string
     telephone?: string
-    bassAccepted?: 'Yes' | 'Unavailable'
+    bassAccepted?: 'Yes' | 'Unavailable' | 'Unsuitable'
```

Comment:
@bell-pa could you check this? it seems to be referred to in a couple of places in code.
Obviously the code could be wrong and it might never be possible to set this value in the DB!